apps/kg/dataloader/KGDataset.py

```diff
@@ -388,18 +411,14 @@ def __init__(self, path, name, files, format, read_triple=True, only_train=False
                                                os.path.join(path, files[1]),
```

Comment:
If you delete the only_train flag. we should force len(files) == 5. User should provide valid and test.

Reply:
Yes, I will add a assert() here